server/serviceClients/authClient/index.js

```diff
@@ -1 +1,17 @@
-export * from './ucp'
+import {AuthClient as UcpAuth} from './ucp'
```

Comment:
It's confusing to import the same thing and name it two different things. This had me looking in the UCP file wondering what was different about UcpAuth vs SophtronAuth

Reply:
they are not the same thing, they are 2 non-related providers that happens to look similar

Reply:
I think you meant to import from sophtron
```javascript
import { AuthClient as SophtronAuth } from './sophtron'
```

Reply:
this is fixed